clevertap-core/src/main/java/com/clevertap/android/sdk/CTInboxMessageListener.java

```diff
@@ -0,0 +1,10 @@
+package com.clevertap.android.sdk;
+
+import com.clevertap.android.sdk.inbox.CTInboxMessage;
+
+public interface CTInboxMessageListener {
+    /**
+     * callback to transfer payload when inbox button is clicked
```

Comment:
@root-ansh-ct javadoc for this says when inbox button is clicked. This is wrong I guess.